ldk/javascript/examples/self-test-loop/src/tests/index.ts

```diff
@@ -13,6 +13,7 @@ import {
 } from '@oliveai/ldk';
 
 import { Cancellable } from '@oliveai/ldk/dist/cancellable';
+import { Urgency } from '../../../../dist/whisper';
```

Comment:
will it be better to import this from @oliveai/ldk/dist/whisper to make sure it is properly exported?

Reply:
Probably! I think the editor helped me out there. I'll update that.